bin/check_tron_jobs.py

```diff
@@ -189,6 +204,12 @@ def check_job_result(job, client, dry_run):
         log.info(result)
     else:
         log.debug("Sending event: {}".format(result))
+        if 'runbook' not in result:
+            log.info("No runbook specified")
+            result['runbook'] = 'foo'
```

Comment:
Say `No runbook specified. Please specify a runbook in the monitoring section of the job definition.`

No need to log